lib/src/sv_bu_list.c

```diff
@@ -21,14 +21,13 @@
 #include <assert.h>
 #ifdef SIGNED_VIDEO_DEBUG
 #include <stdio.h>  // printf
-
-#include "sv_internal.h"  // SHA_HASH_SIZE
 #endif
 #include <stdint.h>
 #include <stdlib.h>  // calloc, malloc, free, size_t
 #include <string.h>  // memcpy
 
 #include "sv_bu_list.h"
+#include "sv_internal.h"
```

Comment:
Remove this include. It is already included in "sv_bu_list.h" which is the main header file for this c-file.